secrets/fake/fake.go

```diff
@@ -0,0 +1,40 @@
+package fake
+
+import (
+	"errors"
+	"github.com/libopenstorage/openstorage/secrets"
+)
+
+var (
+	// ErrNotImplemented default secrets in OSD
+	ErrNotImplemented = errors.New("Not Implemented")
+)
+
+type Fake struct {
+	secrets.SecretManager
```

Comment:
you don't need to add this
it will automatically implement the SecretManager interface if you implement all the methods